Troubleshoot: WebView rewrite, minor changes (#524)
tachiyomiorg/website: src/help/guides/troubleshooting-problems.md

1. Once done, press the *Back Arrow* at the top left to return.
:::

## Update WebView

> To update **WebView**, you need to find what **WebView implementation** is used on your device. In general, default implementation depends on Android version as follows:

Reviewer: ```suggestion
To update **WebView**, you need to find what **WebView implementation** is used on your device. In general, default implementation depends on the Android version as follows:
```

* Android 10.0 and higher - [Android System WebView](https://play.google.com/store/apps/details?id=com.google.android.webview).
* Android from 7.0 to 9.0 - [Google Chrome](https://play.google.com/store/apps/details?id=com.android.chrome).
* Android 6.0.1 and lower - [Android System WebView](https://play.google.com/store/apps/details?id=com.google.android.webview).

::: c-tip
Users of Android 7.0 and higher can also see and change **WebView implementation** in [Developer Options](https://developer.android.com/studio/debug/dev-options).
:::

::: c-warning

> Non-standard implementations like Firefox can lead to incorrect work or crash of Tachiyomi. We recommend to set **WebView implementation** to [Android System WebView](https://play.google.com/store/apps/details?id=com.google.android.webview) or [Google Chrome](https://play.google.com/store/apps/details?id=com.android.chrome).

Reviewer: A bit hard to read, maybe something like this
```suggestion
Non-standard implementations like Firefox can lead to Tachiyomi not working correctly or in the worst-case crash. We recommend to set **WebView implementation** to [Android System WebView](https://play.google.com/store/apps/details?id=com.google.android.webview) or [Google Chrome](https://play.google.com/store/apps/details?id=com.android.chrome).
```
